Configuration/TCA/Overrides/pages.php

```diff
@@ -54,6 +54,16 @@
 $customPageDoktype = 116;
 $customIconClass = 'tx_examples-archive-page';
 
+// Add page type to system
+$dokTypeRegistry = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\DataHandling\PageDoktypeRegistry::class);
```

Comment:
Any particular reason why in pages TCA file instead of ext_tables.php (like before)?

Reply:
@brotkrueml, it seems @calien666 is not working on it anymore, how to proceed?

Reply:
No active reason for doing so. Just remembered on the slack discussion. Rewinded to ext_tables